setup.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
```

Comment:
`setup.py` jest jednym (hehe) z standardowych podejść tworzenia paczek pythonowych. Jest też chyba najstarszy i oficjalny sposób, choć jest ułomny (przez co są inne, lepsze sposoby). Nie musisz więcej wiedzieć niż to że istnieje i że jest tam jakaś magia - w razie problemów można bardziej się wgłębić, ale to jest raczej coś co się robi raz i się potem zapomina